test/parallel/test-event-emitter-add-listeners.js

```diff
@@ -68,8 +68,8 @@ const EventEmitter = require('events');
 }
 
 {
-  const listen1 = function listen1() {};
-  const listen2 = function listen2() {};
+  const listen1 = () => {};
```

Comment:
@Trott on the other hand, `test/parallel/test-event-emitter-remove-all-listeners` relies on `common.noop` being the same object (discovered that by actually making `common.noop` a getter).